finagle-netty4/src/main/scala/com/twitter/finagle/netty4/channel/ServerBridge.scala

```diff
@@ -15,6 +17,17 @@ private[netty4] class ServerBridge[In, Out](
 
   def initChannel(ch: Channel): Unit = {
     val transport: Transport[In, Out] = transportFac(ch)
-    serveTransport(transport)
```

Comment:
Oh I think we can delay `channelRegistered` then given that `channelInit` is [just a fancy version of it](https://github.com/netty/netty/blob/4.1/transport/src/main/java/io/netty/channel/ChannelInitializer.java#L73-L84). What do you think?